content/sensu-go/5.15/reference/etcdreplicators.md

```diff
@@ -61,12 +61,15 @@ example      | {{< highlight shell >}}api_version: federation/v1{{< /highlight >
 
 metadata     |      |
 -------------|------
-description  | Top-level scope that contains the replicator `name`.
+description  | Top-level scope that contains the replicator `name`, `ca_cert`, `cert`, and `key`.
```

Comment:
These fields are not apart of the metadata.

Reply:
I updated them.